Chapters/04-Decomposition-and-ETS.Rmd

```diff
@@ -1,18 +1,18 @@
 # From time series components to ETS {#tsDecomposition}
 
-A very important topic that we need to discuss before we move to the state space models, ETS, ARIMA and other things, is the time series decomposition and the related to it ETS taxonomy. These topics lie in the core of ETS models and are essential for the understanding of the further material.
+Before we turn to state space models, ETS, ARIMA and other things we need to discuss time series decomposition and the ETS taxonomy. These topics lie at the heart of ETS models and are essential for the understanding of the further material.
 
-In this chapter, we start from the discussion of time series components, then move to the idea of decomposing time series in order to see them and then move to the conventional ETS taxonomy, as formulated by @Hyndman2008b, demonstrating its connection with the previous topics.
+In this chapter we start with a discussion of time series components, then move to the idea of decomposing time series into distinct components and then move to the conventional ETS taxonomy, as formulated by @Hyndman2008b, demonstrating its connection with the previous topics.
 
 ## Time series components {#tsComponents}
 The main idea behind many forecasting techniques is that any time series can contain several unobservable components, such as:
 
 1. Level of the series - the average value for specific period of time,
 2. Growth of the series - the average increase or decrease of the value over a period of time,
-3. Seasonality - a pattern, which is observed from year to year (e.g. growth in sales of lager beer in Summer),
-4. Error - an unexplainable white noise.
+3. Seasonality/Periodicity - a pattern, which is observed repeatedly, e.g. over the course of every day (e.g. more sunshine in, well, daylight hours than at night), week to week or year to year (e.g. sales of lager beer in summer),
+4. Error - unexplainable white noise.
```

Comment:
Maybe:
Seasonality - a pattern, repeating itself with a fixed periodicity. For example, we can have hour of day seasonality: every day we have more sunshine in the afternoon than at night. Or we can have month of year seasonality: lager beer is typically sold more in Summer than in Winter.

Reply:
How about

> Seasonality - A pattern that repeats itself. This pattern need not literally be seasonal -- e.g. beer sales being higher every summer than they are in winter. Any pattern with a fixed periodicity works: sales are higher on the first of the month when people get their paycheck (period: monthly) and higher during daytime than they are at night (period: daily).

Reply:
I prefer "day of week", "hour of day" etc, as these terms are not ambiguous. Different people mean different things, when they say "monthly" seasonality. Also, "fixed periodicity" or something like that is important in the beginning of the sentence in my opinion, otherwise it gets lost a bit. Finally, I don't like the example with paycheck, as this look more like an external event to me rather than seasonal pattern - this only happens once per month, all the other days will be different. But this is my personal opinion, I know people who would not agree with me :).
So, another option:

> Seasonality - a pattern that repeats itself with a fixed periodicity. This pattern need not literally be seasonal, like beer sales being higher every summer than they are in winter (season of year). Any pattern with a fixed periodicity works: the number of hospital visitors is higher on Mondays rather than on Saturday or Sunday, because people tend to stay at home over the weekend (day of week seasonality), and sales are higher during daytime than they are at night (hour of day seasonality).

Reply:
Sorry for torturing you, by the way :). If you think that this is too much from my side, please let me know. It might be easier for me just to pull changes and amend, so that you don't need to spend too much time on that. Otherwise, I enjoy the discussion :)

Reply:
No worries. You make good points and the quality gradient is soundly positive. Happy to be tortured if this is torture!

Conversely, if at some point you feel like my edits are getting too aggressive, feel free to just tell me to get lost! :)

Changed to your proposal with minimal edits for style